frontend/src/components/Stats/Stats.css

```diff
@@ -0,0 +1,74 @@
+/*
+Source code for the Substrate Telemetry Server.
+Copyright (C) 2022 Parity Technologies (UK) Ltd.
+
+This program is free software: you can redistribute it and/or modify
+it under the terms of the GNU General Public License as published by
+the Free Software Foundation, either version 3 of the License, or
+(at your option) any later version.
+
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with this program. If not, see <https://www.gnu.org/licenses/>.
+*/
+
+.Stats {
+  text-align: center;
+  padding-top: 2.5rem;
+  padding-bottom: 0.1rem;
+}
+
+.Stats-category {
+  text-align: left;
+  background-color: #fff;
+  margin-bottom: 2.5rem;
```

Comment:
I'm not a huge fan of the thick black margin between each category, but I'm also not bothered enough to suggest an alternative! It'll be easier to see what it's like I think when it's deployed with real node data!